tmx_object.go

```diff
@@ -1,5 +1,6 @@
 /*
 Copyright (c) 2017 Lauris Bukšis-Haberkorns <lauris@nix.lv>
+Copyright (c) 2018 Dominik Madarász <zaklaus@madaraszd.net>
```

Comment:
Could you instead add ` and contributors` as it is not feasible to list every contribution author in copyright line?

Reply:
Oh sure, will do.